pkg/operation/botanist/kubeapiserver.go

```diff
@@ -54,7 +86,8 @@ func (b *Botanist) DefaultKubeAPIServer(ctx context.Context) (kubeapiserver.Inte
 	}
 
 	var (
-		apiServerConfig = b.Shoot.GetInfo().Spec.Kubernetes.KubeAPIServer
+		apiServerConfig   = b.Shoot.GetInfo().Spec.Kubernetes.KubeAPIServer
+		kubernetesVersion = b.Shoot.GetInfo().Spec.Kubernetes.Version
```

Comment:
You can use `b.Shoot.KubernetesVersion` which also is `*semver.Version`, then you don't need to pass it again and again.